packages/react/src/components/Button/Button.Skeleton.tsx

```diff
@@ -9,21 +9,33 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import cx from 'classnames';
 import { usePrefix } from '../../internal/usePrefix';
+import { ButtonSize } from './Button';
 
-const ButtonSkeleton = ({
+export interface ButtonSkeletonProps extends React.HTMLAttributes<HTMLElement> {
+  /**
+   * Optionally specify an href for your Button to become an `<a>` element
+   */
+  href?: string;
+
+  /**
+   * Specify the size of the button, from a list of available sizes.
+   */
+  size?: ButtonSize;
+}
+
+const ButtonSkeleton: React.FC<ButtonSkeletonProps> = ({
   className,
-  small = false,
   href,
   size = 'lg',
   ...rest
-}) => {
+}: ButtonSkeletonProps) => {
   const prefix = usePrefix();
 
   const buttonClasses = cx(className, {
     [`${prefix}--skeleton`]: true,
     [`${prefix}--btn`]: true,
-    [`${prefix}--btn--sm`]: small || size === 'sm',
-    [`${prefix}--btn--md`]: size === 'field' || size === 'md',
```

Comment:
This could possibly be a breaking change, even if it should have been deprecated before now? thoughts @tay1orjones ?

Reply:
@tay1orjones can you take a look?

Reply:
hey @lewandom Taylor is OOO until next week - we'll make sure to take a look early next week :)

Reply:
@lewandom Hey sorry for the delay! Yeah unfortunately we'll need to leave `small` in for now and mark it as deprecated. 

The removal of the check for `field` should be fine though, that's no longer an accepted value for `size`

Reply:
@tay1orjones restored, please check.